WebRoot/templates/mail/Message.template

```diff
@@ -457,6 +468,28 @@
 			</td>
 		</tr>
 	</table>
+
+	<div class='CompDragArea' id='${compDragAreaId}'>
+	<div class='AttachmentImgContainer'>
+		<$= AjxImg.getImageHtml({
```

Comment:
can you pls align the closing div's, "imageName" starting , brackets and all ? 
I see all divs closing in same line.   
AjxImg tag blocks starting at line 474 & 485 to be aligned differently which could be same in alignment with tabs/spaces.